ui/package-lock.json

```diff
@@ -199,7 +199,7 @@
     "@types/jasmine": {
       "version": "2.5.53",
       "resolved": "https://registry.npmjs.org/@types/jasmine/-/jasmine-2.5.53.tgz",
-      "integrity": "sha1-TgzvrQnfXsSMjdQEM1EvhLFWjWE=",
+      "integrity": "sha512-2YNL0jXYuN7w07mb1sMZQ6T6zOvGi83v8UbjhBZ8mhvI1VkQ2STU9XOrTFyvWswMyh5rW1evS+e7qltYJvTqPA==",
```

Comment:
FYI, this seems to be a bug in NPM -- it's picking sha1 or sha512, and uses a different one between Mac and Linux. https://github.com/npm/npm/issues/17749